lib/capybara/session.rb

```diff
@@ -370,18 +369,18 @@ def within_table(locator)
 
     ##
     #
-    # Switch to the given frame
+    # Switch to the given frame.
     #
     # If you use this method you are responsible for making sure you switch back to the parent frame when done in the frame changed to.
-    # Capybara::Session#within_frame is preferred over this method and should be used when possible.
+    # {#within_frame} is preferred over this method and should be used when possible.
     # May not be supported by all drivers.
     #
     # @overload switch_to_frame(element)
-    #   @param [Capybara::Node::Element]  iframe/frame element to switch to
-    # @overload switch_to_frame(:parent)
-    #   Switch to the parent frame
-    # @overload switch_to_frame(:top)
-    #   Switch to the top level document
+    #   @param [Capybara::Node::Element] element    iframe/frame element to switch to
+    # @overload switch_to_frame(frame)
+    #   @param [Symbol] frame    frame/document element to switch to
+    #                            * :parent - the parent frame
+    #                            * :top - the top level document
```

Comment:
[note] Because `:parent` and `:top` are broken, I unify to the one `@overload` definition. See below ⬇️ 

## Before

<img width="611" alt="image" src="https://user-images.githubusercontent.com/473530/58382255-16f62c80-8003-11e9-94e0-5fac6d37972b.png">

https://www.rubydoc.info/gems/capybara/3.21.0/Capybara/Session#switch_to_frame-instance_method

## After

<img width="658" alt="image" src="https://user-images.githubusercontent.com/473530/58382250-0cd42e00-8003-11e9-8fee-d30f67145b4f.png">